2018-03-03T03:12:49.198Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Object with ID 'test1' in collection with ID 'Participant:org.krow.model.User' does not exist)
    at /home/tuckers/.nvm/versions/node/v8.9.4/lib/node_modules/composer-rest-server/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-03-03T03:12:49.200Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Object with ID 'test1' in collection with ID 'Participant:org.krow.model.User' does not exist)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Object with ID 'test1' in collection with ID 'Participant:org.krow.model.User' does not exist)\n\n    at _initializeChannel.then.then.then.then.catch (/home/tuckers/.nvm/versions/node/v8.9.4/lib/node_modules/composer-rest-server/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-03-03T03:13:10.319Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Object with ID 'test1' in collection with ID 'Participant:org.krow.model.User' does not exist)
    at /home/tuckers/.nvm/versions/node/v8.9.4/lib/node_modules/composer-rest-server/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-03-03T03:13:10.319Z ERROR   HLFConnection           :queryChainCode()         {"message":"Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID 'test1' in collection with ID 'Participant:org.krow.model.User' does not exist)","stack":"Error: Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID 'test1' in collection with ID 'Participant:org.krow.model.User' does not exist)\n    at channel.queryByChaincode.then.catch (/home/tuckers/.nvm/versions/node/v8.9.4/lib/node_modules/composer-rest-server/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:809:34)\n    at <anonymous>"}$
2018-03-03T03:13:23.990Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: TypeError: cannot read property 'hasEducation' of undefined)
    at /home/tuckers/.nvm/versions/node/v8.9.4/lib/node_modules/composer-rest-server/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-03-03T03:13:23.990Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: TypeError: cannot read property 'hasEducation' of undefined)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: TypeError: cannot read property 'hasEducation' of undefined)\n\n    at _initializeChannel.then.then.then.then.catch (/home/tuckers/.nvm/versions/node/v8.9.4/lib/node_modules/composer-rest-server/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-03-03T03:13:37.693Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'test1' as the object already exists)
    at /home/tuckers/.nvm/versions/node/v8.9.4/lib/node_modules/composer-rest-server/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-03-03T03:13:37.693Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'test1' as the object already exists)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'test1' as the object already exists)\n\n    at _initializeChannel.then.then.then.then.catch (/home/tuckers/.nvm/versions/node/v8.9.4/lib/node_modules/composer-rest-server/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
